src/lib/header-bidding/prebid/bid-config.ts

```diff
@@ -117,13 +117,37 @@ const getTrustXAdUnitId = (
 	}
 };
 
-const getIndexSiteId = (): string => {
+/**
+ * We store a mapping of sections to Index site ids server-side, where each is
+ * split out by breakpoint. These are transferred to the client via the window,
+ * and read here
+ *
+ * This appears to be an old method of assigning site ids, with the newer method
+ * being to assign them according to ad size (@see getIndexSiteId)
+ */
+const getIndexSiteIdFromConfig = (): string => {
 	const site = window.guardian.config.page.pbIndexSites.find(
 		(s: PrebidIndexSite) => s.bp === getBreakpointKey(),
 	);
 	return site?.id ? site.id.toString() : '';
 };
 
+const getIndexSiteId = (slotSizes: HeaderBiddingSize[]) => {
+	// The only prebid compatible size for fronts-banner-ads and the merchandising-high is the billboard (970x250)
+	// This check is to distinguish from the top-above-nav slot, which includes a leaderboard
+	if (containsBillboardNotLeaderboard(slotSizes)) {
+		return '983842';
+	}
+
+	// Return a specific site id for the mobile sticky slot
+	if (containsMobileSticky(slotSizes)) {
+		return '1047869';
+	}
```

Comment:
Splitting into 2 separate functions seems sensible, clearly shows the logic of getting the site id. I suspect much easier to unpick and further ad to in future